package.json

```diff
@@ -5,6 +5,11 @@
 	"license": "MIT",
 	"repository": "sindresorhus/ky",
 	"funding": "https://github.com/sindresorhus/ky?sponsor=1",
+	"main": "./umd.js",
```

Comment:
This will definitely break all the things. I think https://github.com/sindresorhus/ky/pull/244/files is a much better solution.

Reply:
Ok! will it work for webpack 4 for example?

Reply:
why it will break all the things?

Reply:
Because `ky` is an ESM package so the "main" export should be just that.